common/src/main/java/com/sequenceiq/cloudbreak/telemetry/metering/MeteringConsumptionConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.telemetry.metering;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
i think you can use `@Component` here + also it's not really required to extends from MeteringConfiguration as you duplicationg the config names. you can just  extends from AbstractDatabusConfiguration and inject the metering configuration here (as final field), and set enabled/worker/queue values from the object in the constructor

Reply:
done